simpleflow/swf/helpers.py

```diff
@@ -97,9 +97,14 @@ def find_activity(history, scheduled_id=None, activity_id=None, input=None):
     """
     Finds an activity in a given workflow execution and returns a callable,
     some args and some kwargs so we can re-execute it.
+
+    :type history: simpleflow.history.History
+    :type scheduled_id: str
+    :type activity_id: str
+    :type input: Optional[dict[str, Any]]
     """
     found_activity = None
-    for _, params in history._activities.items():
+    for _, params in history.activities.iteritems():
```

Comment:
The good new is, I've got a bunch of py3-compatibility commits just ready to be pushed.

The bad new is, they're not complete :slightly_frowning_face:... And I went on using the free function `iteritems` instead of `dict.items`.

Mmm, is this method only used by `activity_rerun`? Really no performance optimization to do then O_o